src/Timestampable/Traits/TimestampableEntity.php

```diff
@@ -53,7 +53,7 @@ public function setCreatedAt(\DateTime $createdAt)
     /**
      * Returns createdAt.
      *
-     * @return \DateTime
+     * @return \DateTime|null
```

Comment:
If we are going to allow `null` as return type for these methods, I guess we should first update the hints for the properties that are being returned here.